backend/apps/medical_catalog/api/views/hospital_item_price_views.py

```
@@ -24,26 +25,28 @@ class HospitalItemPriceViewSet(viewsets.ModelViewSet):
 
     def create(self, request, *args, **kwargs):
```

Ruff (0.14.3)

Unused method argument: `args`

(ARG002)